### DISCUSSION TITLE
useFetch interceptors

### DISCUSSION THREAD

**Discussion by danillo-hilbs** [nuxt/nuxt#17264] [Questions] [answered]
How can i intercept/handle errors on useFetch ?
![image](https://user-images.githubusercontent.com/34604307/179584503-4e9feb87-1e99-4e0a-a678-578a42bb8b28.png)
![image](https://user-images.githubusercontent.com/34604307/179584686-d470dbec-21dd-4059-b9d8-bc4355faaa99.png)

**Comment by Flanker72** [comment, marked as answer]
To fix this problem just make the definition async like this:
```ts
return await useFetch(url, {
  ...options,
  onRequestError: async (ctx) => {
    console.log('error: ', ctx);
  },
});
```
